src/Missings.jl

```diff
@@ -2,6 +2,9 @@ __precompile__(true)
 module Missings
 
 import Base: *, <, ==, !=, <=, !, +, -, ^, /, &, |, xor
+if VERSION >= v"0.7.0-DEV.2575"
+    import Dates
```

Comment:
I guess `using` should be enough, right?

Reply:
It should, but `import` is safer since `Base.Dates` used to export only certain names. I believe they should be the same now that it moved to stdlib.

Reply:
No, it's not safer, as it allows overriding functions in `Dates`, which is not what we want to do AFAIK.

Reply:
It depends on the style. `using` has no overriding allowed, but floods the namespace (especially with packages that export many functions and structs) and can cause issues if multiple packages have the same functions. `import` allows to override (though you still have to do it explicitly), but keeps the namespace clear. There is also a warning if any package overrides a method. I always use the module prefix (except with `Base`) since I saw it in an old Julia blog. Either one would do and probably Compat will probably check for the optimal solution.

Reply:
`using` also allows specifying which objects you want to use.